README.md

```diff
@@ -1 +1,30 @@
-# AwesomeBook-ES6
\ No newline at end of file
+# Awesome books: with ES6
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Project
+
+This site was built using [GitHub Pages](https://ismailakinkunmi.github.io/AwesomeBook-ES6/)
+
```

Comment:
- [OPTIONAL] Please add Instructions on how to clone and use the section to your Readme file. This would help users a lot when using your project